### PR TITLE
add net.launchpad.gmult

### PR DESCRIPTION
**Pull request by mikix**
### Multiplication Puzzle

Just a simple little logic game.

The current packaging is complicated by libadwaita not being available in the GNOME 41 Sdk. It will look much simpler once 42 releases.

(Unrelated to this PR, thanks for all you folks do at flathub!)

Homepage: https://launchpad.net/gmult

### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: net.launchpad.gmult.yaml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+app-id: net.launchpad.gmult
+runtime: org.gnome.Platform
+runtime-version: '41'
+sdk: org.gnome.Sdk
+command: gmult
+
+finish-args:
+  - --device=dri # gtk4 wants gl access
+  - --socket=fallback-x11
+  - --socket=wayland
```
Comment (hfiguiere) [inline net.launchpad.gmult.yaml:15]: `--share=ipc` is still required

**Review by hfiguiere — COMMENTED** [review]
you don't need these license files.

File: net.launchpad.gmult.yaml (hunk 1 above)
Comment (mikix) [reply net.launchpad.gmult.yaml:16]: Added! I wondered about that. Everything seemed to still be working fine in x11 without it, but the docs did say x11 would need it.

**Comment by mikix** [comment]
> you don't need these license files.

There are files here released under a license, so I also added the license files. That feels like standard practice to me. Are you saying you'd prefer them be removed?

I like to follow the REUSE guidelines/tooling to help me not forget to list the license for stuff. Including the licenses is part of that spec. So if you don't feel strongly, I'd prefer to keep em.

**Comment by hfiguiere** [comment]
it's a silent "failure".

**Comment by hfiguiere** [comment]
bot, build net.launchpad.gmult

**Comment by mikix** [comment]
> Build [78776 successful](https://flathub.org/builds/#/builders/32/builds/78776)

Tested this build and it looks right to me!

**Comment by mikix** [comment]
Thank you both!

**Pull request closed without merging**